wiki/Contests/OMC/5/kr.md

```diff
@@ -0,0 +1,79 @@
+---
+tags:
+  - OMC5
+  - 'OMC #5'
+---
+
+# osu! Fan Cafe Mapping Contest #5 
+
+![OMC5 Logo](img/omcbg.jpg)
+
+**osu! Fan Cafe Mapping Contest #5** (***OMC5***)는 한국에서 열리고 한국인만 참가하는 것이 가능한 매핑 컨테스트입니다.
+
+## 대회 일정
+
+| 일정 | 날짜 (UTC+9) |
+| --: | :-- |
+| 신청 | 2019-08-12/2019-08-18 |
+| 제출 | 2019-08-19/2019-09-19 |
+| 심사 | 2019-09-20/2019-09-27 |
+| 결과 발표 | 2019-09-28 |
+
+일정은 여건에 따라 지연될 수 있습니다. 모든 일정은 자정 기준입니다.
+
+## 곡
+
+ 참가자는 주최측에서 준비한 두 곡으로만 맵을 제작하게 됩니다. 
+
+ 1. (to be announced)
+ 2. Cranky - R176
+
+## 대회 규칙
+
+### 규칙
+
+
+- 스탠다드 모드 전용 매핑 컨테스트입니다.
+- **한국인**만 참가 가능합니다. 외국 거주, 교포 2세 등의 사유로 osu! 내에서의 플래그가 대한민국이 아닌 경우, 이미 한국인이라고 널리 알려져 있거나 어떠한 방식으로든 입증만 가능하다면 한국인으로서 참가할 수 있습니다.
+- 참가자 당 한 개의 난이도만 심사 대상이 됩니다. 여러 난이도를 제작했더라도 제출은 하나를 정해서 제출해주셔야 합니다. 
+- 참가자들은 Insane 난이도 이상의 비트맵을 제작해야 합니다. 그보다 낮은 난이도는 심사 대상에서 제외됩니다. 
+- 스토리보드, 동영상은 평가 대상이 아닙니다.
+- [Ranking Criteria](/wiki/Ranking_Criteria)를 되도록이면 준수해주시길 바랍니다.
+- 다음과 같은 행동을 한 것이 적발될 경우 참가 자격이 박탈될 수 있습니다. 
+    - 제출 기간이 끝나기 전에 의도적으로 본인의 비트맵을 유출함(비트맵 업로드와 스크린샷 포스트 등이 해당됩니다.) 
+    - 다른 사람에게 히트사운딩을 맡기거나 콜랍을 하는 등의 공동작업
+    - 심사위원을 매수함 
+    - 대회와의 연관성은 관계 없이 물의를 빚어 큰 이슈가 됨 
+    - 상식에서 벗어난 행동을 했고 그 행동이 대회 진행에 영향을 준다고 판단됨
+
+### Division System
+
+
+ 참가자들은 아래 조건에 따라 'Professional'과 'Rookie' 두 진영으로 나뉘어 배정됩니다. 
+- 본인이 호스팅한 랭크 비트맵셋 혹은 Insane 난이도 이상의 게스트 or 콜랍 랭크 비트맵이 존재할 경우 Professional 그룹에서 경쟁하게 됩니다. 
+- 본인이 호스팅한 랭크 비트맵셋 혹은 Insane 난이도 이상의 게스트 or 콜랍 랭크 비트맵이 존재하지 않을 경우 Rookie 그룹에서 경쟁하게 됩니다. 
+- 위의 두 항목은 제출 기간의 시작일을 기준으로 합니다. 
+
+기본적으로 경험이 부족한 신인 매퍼들의 참가를 독려하기 위한 장치라고 생각하시면 편하실거라 생각합니다. 초보 고수 가릴 것 없이 다들 참가해서 재미도 얻고 가능하면 상품까지 가져가는 결과를 얻으시길 바랍니다. 
+## 
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 30,000원 + 프로필 뱃지(프로&루키 둘 다) |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 20,000원 |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 10,000원 |
+
+## 주최
+
+OMC5는 한국의 다음과 같은 매핑 커뮤니티 인원에 의해 주최됩니다.
+
+| 직책 | 인원 |
+| :-- | :-- |
+| 주최 | **[Enon](https://osu.ppy.sh/users/2043401)**, [Beomsan](https://osu.ppy.sh/users/3626063), [GimBab](https://osu.ppy.sh/users/3141964), [My Angel RangE](https://osu.ppy.sh/users/6336713), [Sonnyc](https://osu.ppy.sh/users/11771) |
+| 심사 | [404 AimNotFound](https://osu.ppy.sh/users/2688581), [Beomsan](https://osu.ppy.sh/users/3626063), [eINess](https://osu.ppy.sh/users/3789302), [HabiHolic](https://osu.ppy.sh/users/1974436), [jieusieu](https://osu.ppy.sh/users/759439), [Len](https://osu.ppy.sh/users/1686145), [Sonnyc](https://osu.ppy.sh/users/11771) |
+| 디자인 | [Enon](https://osu.ppy.sh/users/2043401) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/937258)
```

Comment:
"Discussion thread" should be translated (the equivalent of "forum post" is fine too if that's more common)

Reply:
> "Discussion thread" should be translated (the equivalent of "forum post" is fine too if that's more common)

Thanks! Committed.